debian/rules

```diff
@@ -7,3 +7,6 @@
 
 override_dh_auto_test:
 	./runtests pep8 small
+
+override_dh_installman:
+	help2man -o debian/ubuntu-make.1 umake
```

Comment:
Ah, this would work only when umake is installed on your system, (but you will print previous version of umake).

However, some things needs tweaking:
- the build fail if ubuntu make isn't installed. Inded, you need to give the path of git umake (bin/umake) and not the system one (otherwise, you would print previous installed verison man page). Check that this path doesn't shows up in the man page (I did, it doesn't seem so) :)
- the man page is called "ubuntu-make" and not the command name "umake". I guess this is simply a flag in help2man
- when running lintian on the built package (with "gbp buildpackage" to build the package from git tree) it prints some lintian warnings:
W: ubuntu-make: manpage-has-useless-whatis-entry usr/share/man/man1/ubuntu-make.1.gz
W: ubuntu-make: manpage-has-errors-from-man usr/share/man/man1/ubuntu-make.1.gz 87: warning [p 2, 1.5i]: cannot adjust line
Maybe those can be fixed with some help2man flags as well?
Also, after a release, the version printed in the man page is <upstream-version>+unknown (this is because we run some system prepared version not in the usual path with modify. I suggested that just after calling help2man, you sed the file to remove any "+unknown".

I guess that's the one I spotted after a quick test. Should be easy to fix them, do not hesitate if you need any help (just a note: I'm travelling tomorrow, and will be in the CEST time for a week)!